01_login.php

```diff
@@ -68,6 +69,38 @@
 				color: white;
 
 			}
+			
+			/* If the screen size is 601px wide or more, set the font-size of <div> to 80px */
+			@media screen and (min-width: 701px) {
```

Comment:
added responsive fonts; depending on screen size - helpful for end users with phones etc